packages/fiori-annotation-api/src/xml/writer.ts

```diff
@@ -268,6 +268,40 @@ function handleXmlElementAttributeInserts(element: XMLElement, attributeInserts:
     return edits;
 }
 
+function getTextFragmentIndentation(text: string): number {
+    const lines = text.split('\n');
+    const linesWithTextIndices = lines.map((line, idx) => (line.length > 0 ? idx : -1));
+    const indentTextIndex = linesWithTextIndices.find((idx) => idx > -1);
+    if (indentTextIndex == undefined) {
+        return 0;
+    }
+    const indentLineText = lines[indentTextIndex];
+    return Math.ceil(indentLineText.length - indentLineText.trimStart().length) / 4;
```

Comment:
ceil should be calculated after division by 4
Would be nice to have a test case for it

Reply:
done